src/libstd/prelude/v1.rs

```diff
@@ -123,3 +123,6 @@ pub use crate::string::{String, ToString};
 #[stable(feature = "rust1", since = "1.0.0")]
 #[doc(no_inline)]
 pub use crate::vec::Vec;
+#[unstable(feature = "hashmap_prelude", issue = "0")]
```

Comment:
@petrochenkov Does this have any effect? Probably not?

Reply:
It has effect on documentation.
(It will also have effect if someone bothers to move stability checking to its proper place - resolve.)